lldb/test/API/commands/register/register/register_command/TestRegisters.py

```diff
@@ -659,3 +659,16 @@ def test_fs_gs_base(self):
             pthread_self_val.GetValueAsUnsigned(0),
             "fs_base does not equal to pthread_self() value.",
         )
+
+    def test_process_must_be_stopped(self):
+        """Check that all register commands error when the process is not stopped."""
+        self.build()
+        exe = self.getBuildArtifact("a.out")
+        pid = self.spawnSubprocess(exe, ["wait_for_attach"]).pid
+        self.setAsync(True)
+        self.runCmd("process attach --continue -p %d" % pid)
```

Comment:
shouldn't these lines be swapped? Other than that, this LGTM

Reply:
What I found was that `process attach --continue` would never return if you weren't in async mode. Whether that's a bug in itself I'm not sure.

Reply:
All good then!